research/alert_descriptions/index.md

```diff
@@ -0,0 +1,141 @@
+
+*Quote from participant*
+
+**"I would hope to find info on why it was stuck:  not enough nodes with the correct nodeselector, no available nodes with appropriate resources etc."**
+
+While easier said than done, participants wanted to be taken right to the fix rather than the alert itself.
+
+### Click Maps
+
+The behaviors were similar to confidence ratings: responses depended heavily on the user. However, one obvious case was that the 'View Details' link was often the first click.
+
+*Click map of all clicks recorded*, *green-to-red shading indicates lower-higher click density*
+
+![](img/all.png)
+
+This changed somewhat when split out by experience levels, where expert users tended to use the sidebar (navigation menu) slightly more often, as seen by the expanded and multiple green-shaded areas of the dashboard click patterns. This difference was not statistically significant.
+
+*Click map of clicks by experience level*
+
+![](img/all_exp.png)
+
+While interesting to investigate, click maps are imprecise for parsing out statistical patterns in the click data. The data below split out the major sections of the web console interface to compare proportions of click behaviors. The alerts `CPUThrottlingHigh`, `KubeletDown`, `KubePodCrashLooping`, `KubePodNotReady`, and `MachineRunningWithNoPhase` do not have Status (where the 'View Details' button is located) as the most significantly clicked area. This is inferred where the confidence intervals (gray bars) of the purple Status bar overlap with the confidence intervals of another area.
+
+*Proportions of clicks on web console regions by each alert*
+
+![](img/click_area_all.png)
+
+What is it about these alerts where some users are statistically less likely to click 'View Details'?
+
+####  Explaining differences in click locations and what experts do in the web console
+
+In one major theme, "*knowing where to go*", participants indicated that they knew what resource they were looking for or problem they were trying to solve.
+
+*Quotes from participants*
+
+**"I want to see the deployment/deploymentConfig spec to see what the cpu quota setting is set at."**
+
+**"I don't need further details, I want to see more info on the specific pods"**
+
+After mapping out where the comments came from that fit with this theme, every single comment came from one of these alerts where 'View Details' was significantly less likely to be clicked. These data give some evidence that 'View Details' is the default place to go, *unless* the user already knows what they want to do. The user that goes to 'View Details' is likely to be someone who does not know where they want to go next to resolve the alert. This novel insight gave next steps to consider in our design the of 'View Details' page. What does the 'View Details' page currently looks like?
```

Comment:
GitHub's suggestion formatting isn't working well for some reason here, but "design the of" should be "design of the" and "looks like" should be "look like".